.ci/openshift-ci/test.sh

```diff
@@ -21,7 +16,19 @@ oc version || die "Test cluster is unreachable"
 info "Install and configure kata into the test cluster"
 ${script_dir}/cluster/install_kata.sh || die "Failed to install kata-containers"
 
-info "Run test suite: $suite"
-test_status='PASS'
-${script_dir}/run_${suite}_test.sh || test_status='FAIL'
-info "Test suite: $suite: $test_status"
+# Note: let the smoke tests run first and, if failed, do not run the others.
+rc=0
+for suite in "smoke" "e2e"; do
+	info "Run test suite: $suite"
+	test_status='PASS'
+	${script_dir}/run_${suite}_test.sh || test_status='FAIL'
+	info "Test suite: $suite: $test_status"
+	if [ "$test_status" == "FAIL" ]; then
+		rc=1
+		if [ "$suite" == "smoke" ]; then
+			info "Smoke failed, abort the execution of remaining tests"
```

Comment:
I appreciate there are only two suites currently, but maybe you could remove the special treatment for "smoke" and just `break` on error with a call to `info "$suite failed"`?

Reply:
@jodh-intel makes sense. It will then run in "fail fast" mode.